Classes/Middleware/HaikuSeasonList.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
All files should contain the copyright notice: https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/CodingGuidelines/CglPhp/PhpFileFormatting/FileStructure.html#copyright-notice

Reply:
Done, PR for header comment of all other files and adjustment of php-cs-fixer configuration will follow.